src/main/java/online/shop/controller/DeleteUserController.java

```diff
@@ -0,0 +1,23 @@
+package online.shop.controller;
+
+import online.shop.lib.Injector;
+import online.shop.service.UserService;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class DeleteUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("online.shop");
+    private final UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String userId = req.getParameter("id");
+        Long id = Long.valueOf(userId);
+        userService.delete(id);
```

Comment:
I've been advised by our mentors to delete the user's shopping cart when we delete the user :) So maybe you should add the deletion of the shopping cart here)

Reply:
\+